tide_authenticated_content.install

```diff
@@ -132,3 +132,12 @@ function tide_authenticated_content_update_8004() {
     }
   }
 }
+
+/**
+ * Update tide_authenticated_content configurations in database.
+ */
+function tide_authenticated_content_update_8005() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_authenticated_content') . '/config/install'];
+  _tide_import_single_config('tide_authenticated_content.route_settings', $config_location, TRUE);
```

Comment:
for this simple setting, please use ConfigFactory instead of importing config file.